oct_converter/readers/fda.py

```diff
@@ -109,11 +234,18 @@ def read_oct_volume(self):
             volume = np.zeros(
                 (oct_header.height, oct_header.width, oct_header.number_slices)
             )
+
             for i in range(oct_header.number_slices):
                 size = np.fromstring(f.read(4), dtype=np.int32)[0]
                 raw_slice = f.read(size)
-                slice = decode(raw_slice)
+                try:
+                    slice = decode(raw_slice)
+                except:
+                    image = Image.open(io.BytesIO(raw_slice))
+                    slice = np.asarray(image)
```

Comment:
I think we can remove decode here and rely directly on your method. It seems to work well and removes our dependency on `pylibjpeg` which probably causes more errors than any library we depend on.

Reply:
I made the changes and remove the pylibjpeg import and try except lines with decode function.

Reply:
Hi for the second request I can work on that but I need a little bit more time for it.